src/App.css

```diff
@@ -172,3 +177,27 @@ a.home-button svg {
   width: 80%;
   height: 300px;
 }
+
+/* Dictionary Styling */
+.buttons {
```

Comment:
this will be applied to all buttons throughout the app. should we want this for all buttons or the buttons for dictionary?